doc/documentation_data_format.rst

```diff
@@ -157,14 +160,19 @@ Detailed field description
     concentration/amount of that species and will override the initial
     concentration/amount given in the SBML model or given by a preequilibration
     condition. If ``NaN`` is provided for a condition, the result of the
-    preequilibration (or initial concentration/amount from the SBML model, if
+    preequilibration (or initial concentration/amount from the model, if
     no preequilibration is defined) is used.
```

Comment:
Is an SBML assignment rule used if available, or would this have been removed from the model due to `float` values at other positions of the column for this condition entity?

Reply:
The assignment rule should still be used in that case. Good candidate for a new test case.